src/api/java/appeng/api/storage/ISaveProvider.java

```diff
@@ -24,11 +24,19 @@
 package appeng.api.storage;
 
 
+import javax.annotation.Nullable;
+
+
 /**
  * Tells the cell provider that changes have been made an the cell must be persisted
  *
  */
 public interface ISaveProvider
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Annotate the "ISaveProvider" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1609)